src/Miso/Style.hs

```diff
@@ -267,34 +279,14 @@ style_ = MT.Styles . M.fromList
 styleInline_ ::  MisoString -> Attribute action
 styleInline_ = textProp "style"
 -----------------------------------------------------------------------------
--- | 'renderStyleSheet'
---
--- Renders a 'StyleSheet' to a 'MisoString'
---
--- @
--- testSheet :: StyleSheet
--- testSheet =
---   sheet_
---   [ ".name" =:
---       style_
---       [ backgroundColor "red"
---       , alignContent "top"
---       ]
---   , "#container" =:
---       style_
---       [ backgroundColor "blue"
---       , alignContent "center"
---       ]
---   ]
--- @
---
+-- | Renders a 'Styles' to a 'MisoString'
 renderStyles :: Styles -> MisoString
 renderStyles m = MS.unlines
   [ mconcat [ spaced, k, ":", v, ";" ]
   | (k,v) <- M.toList m
-  , let spaced = "  "
+  , let spaced = " "
```

Comment:
Probably the only non-haddock change - let me know if using 2 spaces was intentional (or is it for aesthetic reasons? Seems unnecessary to increase stylesheet size)